binding/java/build.gradle

```diff
@@ -55,18 +56,15 @@ ext.projectArtefactDir      = "jars/"
 ext.projectMainClass        = "org.openqa.selenium.phantomjs.PhantomJSDriver"
 ext.projectCreator          = "Ivan De Marino (https://github.com/detro)"
 
-ext.seleniumVersion         = "2.44.0"
 
 dependencies {
-    ["selenium-java", "selenium-remote-driver"].each {
-        compile "org.seleniumhq.selenium:$it:$seleniumVersion"
-    }
+    compile "org.seleniumhq.selenium:selenium-remote-driver:$seleniumVersion"
 }
 
-// Forces Java 6 compile source/target
+// Forces Java 8 compile source/target
```

Comment:
Good idea. But even a bigger reason for a `2.0.0` release. You are moving bytecode compatibility to Java 8 (and I agree) but in this case a major version jump is necessary.